src/autogluon/bench/cloud/aws/docker/Dockerfile

```diff
@@ -17,5 +27,9 @@ RUN if [ -n "$AG_BENCH_DEV_URL" ]; then \
         && pip install autogluon.bench==$AG_BENCH_VERSION; \
     fi
 
-RUN chmod +x entrypoint.sh
+RUN chmod +x entrypoint.sh /app/gpu_utilization.sh \
+    && echo "*/5 * * * * CDK_DEPLOY_REGION=/app/gpu_utilization.sh >> /var/log/cron.log 2>&1" > /var/spool/cron/crontabs/root \
```

Comment:
So basically we are injecting this into `cron.log`, which will be automatically called to report metrics?

Reply:
That's right. `service cron start` in the `entrypoint.sh` will start the cron server which runs this line according to the schedule. It automatically reports metrics every 5 minutes.

Reply:
It seems to be a pretty standard way to do such kind of thing according to this thread https://stackoverflow.com/questions/37458287/how-to-run-a-cron-job-inside-a-docker-container